### PR TITLE
Update storybook docs for breadcrumbs, calendar and callout

### PR DESCRIPTION
**Pull request by RobertoMSousa**
Fixes callout, breadcrumbs and calendar stories.

**Changes proposed in this pull request:**
Update the docs of the components mentioned previously to allow the storybook 6 to render the docs page and fix a bug on the calendar that was preventing the component to be rendered at all on storybook.

### CODE REVIEW

File: stories/callout.stories.tsx [hunk 1]
```diff
@@ -1,63 +1,116 @@
-import { storiesOf } from '@storybook/react'
+// import { storiesOf } from '@storybook/react'
```
Comment (jackcmeyer) [inline stories/callout.stories.tsx:1]: commented out import

File: stories/calendar.stories.tsx [hunk 2]
```diff
@@ -1,57 +1,61 @@
...
+const end = moment().add(hours, 'hours')
+
+const Template: Story<CalendarProps> = (args) => (
+  <div>
+    <Calendar {...args} />
+    <Toaster autoClose={800} hideProgressBar draggable />
+  </div>
+)
+
+// main story tha's editable and has the docs for the props
```
Comment (jackcmeyer) [inline stories/calendar.stories.tsx:29]: small spelling typo

**Review by jackcmeyer — CHANGES_REQUESTED** [review]

**Review by blestab — APPROVED** [review]
LGTM except for the typo mentioned by @jackcmeyer

**Review by blestab — CHANGES_REQUESTED** [review]
LGTM except for the typo mentioned by @jackcmeyer

**Comment by RobertoMSousa** [comment]
@blestab , typo fixed.
thanks 🙂

**Comment by blestab** [comment]
Thanks @RobertoMSousa 
May you address the commented out import in `stories/callout.stories.tsx` as per @jackcmeyer comment i.e. if we are no longer using it then let's remove it.

**Comment by RobertoMSousa** [comment]
> Thanks @RobertoMSousa
> May you address the commented out import in `stories/callout.stories.tsx` as per @jackcmeyer comment i.e. if we are no longer using it then let's remove it.

That was already removed on t[his commit ](https://github.com/HospitalRun/components/pull/637/commits/b5f80ad7ccd9aa77f36dd1b5b6522a9853032e5c)

**Review by tehKapa — APPROVED** [review]

**Pull request merged**